### PR TITLE
issue-1-set-location - Jianni Cariaga

### PR DESCRIPTION
**Pull request by jiannicariaga**
feature1_Cariaga, Jianni

### CODE REVIEW

File: server/public/styles.css [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+.defenition > h1 {
+  font-size: 3rem;
+  letter-spacing: 1px;
+}
+
+.defenition > h3 {
+  font-size: 1.5rem;
+}
+
```
Comment (thebearingedge) [inline server/public/styles.css]: tpyo

File: server/public/index.html [hunk 2]
```diff
@@ -3,14 +3,20 @@
...
+  <link href="https://fonts.googleapis.com/css2?family=PT+Sans:ital,wght@0,400;0,700;1,400&family=Quicksand:wght@400;700&display=swap" rel="stylesheet">
   <link rel="stylesheet" href="/reset.css">
   <link rel="stylesheet" href="/layout.css">
   <link rel="stylesheet" href="/styles.css">
 </head>
 <body>
   <div id="root"></div>
   <script src="/main.js"></script>
+  <script src="https://kit.fontawesome.com/94fa55cf7c.js" crossorigin="anonymous"></script>
+  <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.2/dist/js/bootstrap.bundle.min.js" integrity="sha384-OERcA2EqjJCMA+/3y+gxIOqMEjwtxJY7qPCqsdltbNJuaOe923+mo//f6V8Qbsw3" crossorigin="anonymous"></script>
```
Comment (thebearingedge) [inline server/public/index.html]: I don't recommend including the bootstrap javascript, you should use react-bootstrap if you need interactive bootstrap components

File: client/pages/home.jsx [hunk 3]
```diff
@@ -1,10 +1,12 @@
...
+import Defentiion from '../components/defenition';
 
 export default function Home(props) {
   return (
-    <div>
-      <HelloWorld />
-    </div>
+    <main className='container'>
+      <SearchForm />
+      <Defentiion />
```
Comment (thebearingedge) [inline client/pages/home.jsx]: tpyo

File: client/components/search-form.jsx [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+    if (event.target.value === '') this.setState({ message: '' });
+    if (this.state.geolocation) {
+      this.setState({
+        geolocation: null,
+        message: 'Device location removed.'
+      });
+    } else {
+      this.setState({ location: event.target.value });
+    }
+  }
```
Comment (thebearingedge) [inline client/components/search-form.jsx]: i could be wrong, but do we need the user's geolocation elsewhere in the app? should we lift the geolocation state up to App?

**Review by thebearingedge — CHANGES_REQUESTED** [review]

File: client/components/search-form.jsx (hunk 4 above)
Comment (jiannicariaga) [reply client/components/search-form.jsx]: Geolocation is only needed for the fetch to Yelp. The response has a "center" property which is the same value which will be used to center the Google Map in feature 3.

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**